### ISSUE TITLE
ADA 2024-04-24 Rule (WCAG 2.1, Level AA) compliance

### ISSUE THREAD

**Issue by wwarriner** [uabrc/uabrc.github.io#967]
We need to ensure our docs are in compliance with the ADA 2024-04-24 rule by 2026-04-24. This means compliance with at least WCAG 2.1, Level AA.

This should be relatively straightforward. Bear in mind one major goal is to make content perceivable to readers with low vision or blindness. Some things to ensure compliance:

- Ensure the presence and quality of alt-text. https://www.wcag.com/blog/good-alt-text-bad-alt-text-making-your-content-perceivable/
- Ensure sufficient contrast ratios in style.
- Ensure sufficient contrast ratios in text-based images.
- Ensure screen readers can perceive text from text-based images by supplying a transcription of the contents in the document body.

And we need to ensure we are checking all of the above during PRs. It would be helpful to have a PR template with a checklist of all the relevant items.

Additional resources and reading:

- Fact Sheet: https://www.ada.gov/resources/2024-03-08-web-rule/
- WCAG Guidelines: https://www.w3.org/WAI/standards-guidelines/wcag/
- Fundamentals Overview: https://www.w3.org/WAI/fundamentals/
